js/src/layers/MarkerCluster.js

```diff
@@ -52,12 +76,35 @@ export class LeafletMarkerClusterView extends layer.LeafletLayerView {
 
   create_obj() {
     var options = this.get_options();
-    this.obj = L.markerClusterGroup(options);
+    this.obj = L.markerClusterGroup({
+      disableClusteringAtZoom: options['disableClusteringAtZoom'],
+      maxClusterRadius: options['maxClusterRadius'],
```

Comment:
Instead of passing options manually you can unroll all of them with the following
```suggestion
    this.obj = L.markerClusterGroup({
      ...options,
```